pkg/storkctl/clusterpair.go

```diff
@@ -73,3 +76,38 @@ func clusterPairPrinter(clusterPairList *storkv1.ClusterPairList, writer io.Writ
 	}
 	return nil
 }
+
+func newGenerateClusterPairCommand(cmdFactory Factory) *cobra.Command {
+	generateClusterPairCommand := &cobra.Command{
+		Use:   clusterPairSubcommand,
+		Short: "Generate a spec to be used for cluster pairing from a remote cluster",
+		Run: func(c *cobra.Command, args []string) {
+			config, err := cmdFactory.RawConfig()
+			if err != nil {
+				handleError(err)
+			}
+
+			clusterPair := &storkv1.ClusterPair{
+				TypeMeta: meta.TypeMeta{
+					Kind:       reflect.TypeOf(storkv1.ClusterPair{}).Name(),
+					APIVersion: storkv1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: meta.ObjectMeta{
+					Name: "<insert_name_here>",
+				},
+
+				Spec: storkv1.ClusterPairSpec{
+					Config: config,
+					Options: map[string]string{
+						"<insert_storage_options_here>": "",
```

Comment:
is the use case to create a template spec which users have to complete?

Reply:
Yes, the template has the kubeconfig populated, they only need to add the storage options